draft-ietf-tls-exported-authenticator.md

```diff
@@ -244,6 +244,14 @@ Where Hash is the hash function negotiated by TLS. If the authenticator request
 is not present, it is omitted from this construction (that is, it is zero
 length).
 
+If the party that generates the exported authenticator does so with a different
+connection than the party that is validating it, then the Handshake Context will
+not match, resulting in a CertificateVerify message that does not validate.
+This include situations in which the application data is sent via TLS-terminating
+proxy.  Given a failed CertificateVerify validation, it may be helpful for
+the application to confirm that both peers share the same connection
+using a value derived from the exporter before taking a user-visible action.
```

Comment:
"the exporter" => "an exporter" or "the connection secrets"; there isn't a single exporter that we're working with here.